.github/copilot-instructions.md

```
@@ -70,26 +70,26 @@ dotnet test Qwiq.sln --configuration Release --settings coverage.runsettings
-| `Qwiq.Core`          | net472;netstandard2.0;net8.0 | Core interfaces and abstractions    |
-| `Qwiq.Core.Rest`     | net472;netstandard2.0;net8.0 | REST API client implementation      |
-| `Qwiq.Core.Soap`     | net472                       | SOAP client (Windows only)          |
-| `Qwiq.Linq`          | net472;net8.0                | LINQ query provider                 |
-| `Qwiq.Mapper`        | net472;net8.0                | Object mapping layer                |
-| `Qwiq.Identity`      | net472;net8.0                | Identity management                 |
-| `Qwiq.Identity.Soap` | net472                       | Identity SOAP client (Windows only) |
+| Project              | Target Frameworks                         | Description                         |
+| -------------------- | ----------------------------------------- | ----------------------------------- |
+| `Qwiq.Core`          | net472;net48;net481;net8.0;net9.0;net10.0 | Core interfaces and abstractions    |
+| `Qwiq.Core.Rest`     | net472;net48;net481;net8.0;net9.0;net10.0 | REST API client implementation      |
+| `Qwiq.Core.Soap`     | net472                                    | SOAP client (Windows only)          |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)